cmd/testdata/gatewayd.yaml

```
@@ -73,3 +73,9 @@ servers:
 
 api:
   enabled: True
+
+raft:
+  address: 127.0.0.1:2223
+  nodeID: node2
```

If this is a bootstrap node, shouldn't it be node1?

```suggestion
  nodeID: node1
```

---

Not necessary, bootstrap is handled by the `isBootstrap` parameter. However, for consistency, it’s better to have node1 even for test case data. Addressed in [ef3745b](https://github.com/gatewayd-io/gatewayd/pull/636/commits/ef3745bc6803fd58014b23e0daffad1e5db68017).